editor/src/messages/tool/tool_messages/select_tool.rs

```diff
@@ -86,6 +109,18 @@ impl PropertyHolder for SelectTool {
 	fn properties(&self) -> Layout {
 		Layout::WidgetLayout(WidgetLayout::new(vec![LayoutGroup::Row {
 			widgets: vec![
+				RadioInput::new(vec![
+					RadioEntryData::new("Layer")
+						.value("layer")
+						.tooltip("Layer")
+						.on_update(move |_| SelectToolMessage::SelectOptions(SelectOptionsUpdate::Type(SelectedType::Layer)).into()),
+					RadioEntryData::new("Group")
+						.value("group")
+						.tooltip("Group")
+						.on_update(move |_| SelectToolMessage::SelectOptions(SelectOptionsUpdate::Type(SelectedType::Group)).into()),
+				])
+				.selected_index((self.tool_data.selected_type == SelectedType::Group) as u32)
+				.widget_holder(),
```

Comment:
Could we have some more context here to make it clear that this is referring to selection behaviour? Some hint to the user about double clicking and control clicking would also be appreciated.

Reply:
Yeah, this should go in the hints bar. But I haven't figured out how to make an icon for double clicking, I should revisit that.

Reply:
@ChrisMend19 When you rebase/merge from master, you can now use `MouseMotion::LmbDouble` as the double click icon for the hints bar. Please resolve this comment when that's done.